content/docs/00-release-notes.md

```diff
@@ -73,6 +73,11 @@ Spectro Cloud Palette 3.0.0 is released with [Native Edge](/clusters/edge/native
 * Open Policy Agent version 3.9.0
 * AWS EBS  CSI version 1.10.0
 
+**Known Issues**
+
+* While deploying multiple applications on a Sandbox cluster, if one of the application get stuck because of some errors it can cause other deployments in the same sandbox cluster to get stuck in deploying state.
```

Comment:
Our UI refers to applications and deployments as **apps**, shouldn't we use the same term in documentation, @tiwarisumit3 @resmaa?

What do you think of the following text?

> While deploying multiple apps in a sandbox cluster, if the deployment of one of the apps is blocked due to errors then subsequent apps deployed to the same sandbox cluster might also be stuck in deploying state.

Reply:
@amarjeet27 We should mention the workaround for this issue. Simply mentioning the issue and not providing the solution can frustrate the reader.

Reply:
@amarjeet27 We should mention the workaround for this issue. Simply mentioning the issue and not providing the solution can frustrate the reader.

Reply:
@karl-cardenas-coding - good point. We can mention something like:

1. Delete the stuck deployment. 
2. Fix the app with the error.
3. Redeploy the app again.